tsconfig.base.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
```

Comment:
we gonna use this also for node packages etc, particular packages should explicitly state their target (in future babel/esbuild will handle that automatically)